src/_index.js

```diff
@@ -94,6 +103,12 @@ class Slider{
         this.isPlay = true;
         checkAutoplay(this);
 
+        // pauseOnHover (priority: attribute > options)
+        if(this.wrapper.hasAttribute(this._attr.pauseOnHover)){
+            const isPauseOnHover = this.wrapper.getAttribute(this._attr.pauseOnHover).toLowerCase().trim();
+            this.options.pauseOnHover = isPauseOnHover === "true";
```

Comment:
Whenever the attribute for `pauseOnHover` appear in the wrapper element, we will have `pauseOnHover: true`. 

Don't care what the value is. 

This is just the same with the `swipe` option which you might have had a look already, haven't you?